packages/ethereum/redux/selectors.js

```diff
@@ -10,28 +10,69 @@ import { DUCK_ETHEREUM } from './constants'
 export const getDuckEthereum = () => (state) =>
   state[DUCK_ETHEREUM]
 
+const getBitcoinWallets = (bitcoin) => {
+  let bitcoinWallets = []
+  for (const key in bitcoin.list) {
+    for (const secKey in bitcoin.list[key]) {
+      bitcoinWallets = [
+        ...bitcoinWallets,
+        {
+          data: [
+            {
+              address: bitcoin.list[key][secKey].address,
+              blockchain: 'BTC',
+            },
+          ],
+          title: 'Bitcoin example',
```

Comment:
Why we need hardcoded data here?
By the way we have no wallets' titles in RN app. We may use wallets' addresses instead